
def app_throws():
    raise RuntimeError("Cannot Continue!!!")
